tmt/base.py

```diff
@@ -2130,6 +2130,21 @@ def lint_unknown_keys(self) -> LinterReturn:
 
         yield LinterOutcome.PASS, 'correct keys are used'
 
+    def lint_empty_env_files(self) -> LinterReturn:
+        """ P001: env files are not empty """
+
+        env_files = self.node.get("environment-file") or []
+        if env_files:
+            for env_file in env_files:
+                env_file = Path(env_file).resolve()
+
+                if not env_file.exists() or not env_file.stat().st_size:
+                    yield LinterOutcome.FAIL, f'the file "{env_file}" does not exists or is empty'
```

Comment:
The non-existent file is actually not covered:

```
tmt lint

File '/tmp/empty/environment' doesn't exist.
```

We do not get the full `lint` output as expected.

Reply:
Meh, incorrectly indented `return`, should be on the level of `for`.

Reply:
I see, moving `return` makes sense. It does not fix the problem mentioned above though.

Reply:
The non-existent case is still not fixed:

```
> tmt lint

File '/tmp/empty/environment' doesn't exist.
```